.github/CODEOWNERS

```diff
@@ -0,0 +1,69 @@
+# <-- DOCS FOR THIS FILE -->
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+# *       @global-owner1 @global-owner2
+
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
+# *.js    @js-owner
+
+# You can also use email addresses if you prefer. They'll be
+# used to look up users just like we do for commit author
+# emails.
+# *.go docs@example.com
+
+# In this example, @doctocat owns any files in the build/logs
+# directory at the root of the repository and any of its
+# subdirectories.
+# /build/logs/ @doctocat
+
+# The `docs/*` pattern will match files like
+# `docs/getting-started.md` but not further nested files like
+# `docs/build-app/troubleshooting.md`.
+# docs/*  docs@example.com
+
+# In this example, @octocat owns any file in an apps directory
+# anywhere in your repository.
+# apps/ @octocat
+
+# In this example, @doctocat owns any file in the `/docs`
+# directory in the root of your repository.
+# /docs/ @doctocat
+# <-- /DOCS FOR THIS FILE -->
+
+# <-- COMMON TO ALL MAINTAINERS -->
+/*.json @publiclab/is-maintainers
+/*.md @publiclab/is-maintainers
+/*.lock @ubliclab/is-maintainers
+/Gruntfile.js @publiclab/is-maintainers
+/.github/ @publiclab/is-maintainers
+# <-- /COMMON TO ALL MAINTAINERS -->
+
+# <-- SPECIFIC MAINTAINERS -->
+/index.js @publiclab/is-cli-maintainers
+/src/cli/ @publiclab/is-cli-maintainers
+/src/CliUtils.js @publiclab/is-cli-maintainers
+
+/src/*.js @publiclab/is-core-maintainers
+/src/*.json @publiclab/is-core-maintainers
+/src/util/ @publiclab/is-core-maintainers
```

Comment:
```suggestion
/src/util/ @publiclab/is-core-maintainers
/test/core/* @publiclab/is-core-maintainers
```

Reply:
so, having test maintainers makes it a bit weird - like, they would take precedence and core maintainers wouldn't get notified here, i think?

Reply:
Ok